client/my-sites/theme/themes-related-card/index.jsx

```diff
@@ -70,4 +71,8 @@ const ThemesRelatedCard = React.createClass( {
 	}
 } );
 
-export default ThemesRelatedCard;
+export default connect(
+	state => ( {
+		getDetailsUrl: getThemeDetailsUrl.bind( null, state )
```

Comment:
This is a matter of preference but this could also be `getDetailsUrl: theme => getThemeDetailsUrl( state, theme )` to get rid of the `bind` and also self document that the method takes a single parameter